library/sha256.c

```diff
@@ -314,7 +350,11 @@ int mbedtls_sha256_update_ret( mbedtls_sha256_context *ctx,
     if( ilen > 0 )
         mbedtls_platform_memcpy( (void *) (ctx->buffer + left), input, ilen );
 
-    return( 0 );
+    if( ilen < 64 )
```

Comment:
I am guessing it is related with `while( ilen >= 64 )` but it is not very clear. Could you add some comment that it is about FI atak

Reply:
Fixed.